package.json

```diff
@@ -86,6 +85,7 @@
   "devDependencies": {
     "@studio/changes": "^2.0.1",
     "eslint": "^7.13.0",
+    "mocha": "^8.4.0",
```

Comment:
It occured to me Mocha might just be a dev dependency here (as the tests should consume the Mocha used by mocaccino). Correct me if I'm wrong.